Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:14.1.0-alpine3.11
+
+WORKDIR /app
+ENV PATH /app/node_modules/.bin:$PATH
+
+COPY package.json ./
+COPY package-lock.json ./
+
+RUN npm install --silent
+RUN npm install react-scripts@3.4.1 -g --silent
```

Comment:
why should we iinstall this?

Reply:
as long as you're having react-script in ur dependencies in ur package.json u dont need this.